configuration/sitemaps.md

```diff
@@ -451,15 +459,17 @@ Splits a String Item at each separator into multiple rows.
 
 Mappings is an optional parameter for the [Switch](#element-type-switch) and [Selection](#element-type-selection) element types.
 
-Please be aware of the fact, that both Switch and Selection are input element types.
-If you are looking to transform Item data into meaningful outputs, a [Text element](#element-type-text) with it's label parameter may be a better choice.
+Please note that Switch and Selection are input-only element types.
+In other words, they only accept commands.
+For more information, see [Commands vs. Status]({{base}}/configuration/items.md#command-vs-status) in the [Items]({{base}}/configuration/items.md) article.
```

Comment:
I might be the author of the original version of this. I have to admit that it is misleading.

A Switch or a Selection can of course also be used to represent the current state. I wanted to highlight, that a text element is recommended IF the underlying Item can not be written (e.g. Time of Day). Not sure how this relates to Commands vs Status.  Maybe remove this paragraph all together?

Reply:
I agree that this paragraph should be deleted.

Reply:
This comment has been addressed.